rviz_rendering/src/rviz_rendering/render_system.cpp

```diff
@@ -330,32 +335,34 @@ RenderSystem::setupResources()
   //   throw std::runtime_error(s);
   // }
 
-// TODO(wjwwood): figure out how to replace/port media export.
-#if 0
-  // Add paths exported to the "media_export" package.
-  std::vector<std::string> media_paths;
-  ros::package::getPlugins("media_export", "ogre_media_path", media_paths);
-  std::string delim(":");
-  for (auto iter = media_paths.begin(); iter != media_paths.end(); ++iter) {
-    if (!iter->empty()) {
-      std::string path;
-      int pos1 = 0;
-      int pos2 = iter->find(delim);
-      while (pos2 != static_cast<int>(std::string::npos)) {
-        path = iter->substr(pos1, pos2 - pos1);
-        ROS_DEBUG("adding resource location: '%s'\n", path.c_str());
-        Ogre::ResourceGroupManager::getSingleton().addResourceLocation(
-          path, "FileSystem", "rviz_rendering");
-        pos1 = pos2 + 1;
-        pos2 = iter->find(delim, pos2 + 1);
+  addAdditionalResourcesFromAmentIndex();
+}
+
+void RenderSystem::addAdditionalResourcesFromAmentIndex() const
+{
+  const std::string RVIZ_OGRE_MEDIA_RESOURCE_NAME = "rviz_ogre_media_exports";
+  std::map<std::string,
+    std::string> resource_locations = ament_index_cpp::get_resources(
+    RVIZ_OGRE_MEDIA_RESOURCE_NAME);
+  for (auto resource : resource_locations) {
+    std::string content;
+    std::string prefix_path;
+    if (ament_index_cpp::get_resource(RVIZ_OGRE_MEDIA_RESOURCE_NAME, resource.first, content,
+      &prefix_path))
+    {
+      std::vector<std::string> filenames =
+        string_helper::splitStringIntoTrimmedItems(
+        content, '\n');
```

Comment:
nitpick: I would restyle this to be:

```c++
      std::vector<std::string> filenames =
        string_helper::splitStringIntoTrimmedItems(content, '\n');
```

Or:

```c++
      std::vector<std::string> filenames = string_helper::splitStringIntoTrimmedItems(
        content, '\n');
```

But not both. You don't need to fix this particular case, but just for future reference.